multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/util/ClientReleaser.java

```diff
@@ -27,10 +27,13 @@ public void releaseClientFor(HistoryService historyService, String processInstan
         String organizationName = HistoryUtil.getVariableValue(historyService, processInstanceId, Variables.ORGANIZATION_NAME.getName());
         String spaceName = HistoryUtil.getVariableValue(historyService, processInstanceId, Variables.SPACE_NAME.getName());
         String spaceGuid = HistoryUtil.getVariableValue(historyService, processInstanceId, Variables.SPACE_GUID.getName());
+        String correlationId = HistoryUtil.getVariableValue(historyService, processInstanceId, Variables.CORRELATION_ID.getName());
 
         try {
-            clientProvider.releaseClient(user, organizationName, spaceName);
-            clientProvider.releaseClient(user, spaceGuid);
+            clientProvider.releaseClient(user, organizationName, spaceName, null);
+            clientProvider.releaseClient(user, spaceGuid, null);
```

Comment:
Can we reduce somehow the invocation of all these functions? I would be easy to forget what and when to release

Reply:
The idea behind this was just to try to release clients created during polling of operation